src/app/public/modules/popover/popover-content.component.html

```diff
@@ -12,20 +12,36 @@
   #popoverRef
 >
   <div
-    class="sky-popover sky-shadow sky-rounded-corners"
+    class="sky-popover sky-shadow"
     tabindex="-1"
+    [ngClass]="{
+      'sky-rounded-corners': themeName === 'default',
+      'sky-elevation-4': themeName === 'modern'
+    }"
   >
     <header *ngIf="popoverTitle"
-      class="sky-popover-header sky-padding-even-default"
+      class="sky-popover-header"
+      [ngClass]="{
+        'sky-padding-even-default': themeName === 'default',
+        'sky-padding-even-lg sky-margin-stacked-lg': themeName === 'modern'
+      }"
     >
       <h1 *ngIf="popoverTitle"
-        class="sky-popover-title sky-emphasized"
+        class="sky-popover-title"
+        [ngClass]="{
+          'sky-emphasized': themeName === 'default',
+          'sky-font-emphasized': themeName === 'modern'
```

Comment:
Did `sky-font-emphasized` come from UX? I'm not seeing it on the [design spec](https://blackbaud.invisionapp.com/console/share/UG1N3XTCC7/485027107)...

Reply:
I discussed that one with @Blackbaud-ToddRoberts. The title portion of the popover is not in the design spec at all. Todd and I synced on the styling for the title and I also showed it to Adam and he was ok with it.

Reply:
Sounds good to me.